appinventor/buildserver/src/com/google/appinventor/buildserver/Compiler.java

```diff
@@ -2273,7 +2310,7 @@ private boolean runAaptPackage(File manifestFile, File resDir, String tmpPackage
       aaptPackageCommandLineArgs.add("--output-text-symbols");
       aaptPackageCommandLineArgs.add(symbolOutputDir.getAbsolutePath());
       aaptPackageCommandLineArgs.add("--no-version-vectors");
-      appRJava = new File(sourceOutputDir, packageName.replaceAll("\\.", "/") + "/R.java");
+      appRJava = new File(sourceOutputDir, packageName.replaceAll("\\.", "/") + SLASH + "R.java");
```

Comment:
It seems like the replaceAll call should also use SLASH to be windows compatible?

Reply:
Yeah, I have actually changed somewhen, but I don't know why it was not pushed. Changed it back.